CHANGELOG.next.toml

```diff
@@ -103,3 +103,36 @@ message = "Upgrade to Smithy 1.26.2"
 references = ["smithy-rs#1972"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "rcoh"
+
+[[smithy-rs]]
+message = "Generate enums that guide the users to write match expressions in a forward-compatible way."
```

Comment:
The details in the `aws-sdk-rust` entry should also be in the `smithy-rs` entry since there are people using smithy-rs to generate clients for their own services.

Reply:
That makes sense. Duplicated the `message` into the `[[smithy-rs]]` section in 04e1a22.